Source/Glass.Mapper.Sc/SitecoreVersionAbstractions.cs

```diff
@@ -36,6 +36,7 @@ public static string GetItemUrl(Item item, UrlOptions urlOptions)
         {
 #if SC104
             if (item?.Paths == null) return string.Empty;
+            if (item?.Paths.FullPath.Contains("[orphan]") && item?.Parent == null) return string.Empty;

```

Comment:
item?.Paths.FullPath is an expensive operation. This also contains magic strings.

Reply:
Issue is coming for orphan items where item.parent is null but we cannot add only this check because for root item also parent is null. So added check for fullpath "[orphan]/renderingParameters"

Reply:
@muso31  Thanks for flagging. Replaced item?.Paths.FullPath check with item?.Paths.IsFullyQualified. This will come false for all orphan items.